hlapipeline/alignimages.py

```diff
@@ -45,7 +49,7 @@
                                       "classify": False,
                                       "threshold": 10},
                                  "wfc":
-                                     {"fwhmpsf": 0.076,
+                                     {"fwhmpsf": 0.076,#0.13
```

Comment:
The PSF FWHM at F550M varies from 0.10 to 0.13 arcseconds as documented in ACS ISR 2007-12 and 2008-03 and reported in the ACS Instrument Handbook pg 83.